FastSurferCNN/download_checkpoints.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 Image Analysis Lab, German Center for Neurodegenerative Diseases (DZNE), Bonn
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+
+from utils.checkpoint import check_and_download_ckpts, get_checkpoints, VINN_AXI, VINN_COR, VINN_SAG, URL
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Check and Download Network Checkpoints')
+    parser.add_argument("--all", default=False, action="store_true",
```

Comment:
I would rather make this a flag with which you can provide different flags (all, vinn, cnn) and use all as the default

Reply:
Currently, vinn should be the only and automatic option. If people want to use faalstsurfercnn, they should use fastsurfer 1.X imo
If we integrate other modules for other task, it should be a list of modules (e.g. aseg, aparc)

Reply:
I want an all flag to download everything that is available. This will be used in docker. 
Integration of fscnn well be decided based on vinn performance in tests.

Reply:
My main issue here is that there should not be multiple flags (--all --vinn --cnn). Especially because at the  moment you need to specify both (--all and --vinn). "all" should already include "vinn" at the very least. Otherwise it is very counterintuitive.

Reply:
all does include vinn, see the fixes below. I agree that we need to think about having a --flag for each module later or having a --module <string or list>. I think there is really no difference here. What is more important is to think about good names for the modules. vinn is not great as many module will use the vinn architecture. Probably should be fsvinn for FastsurferVINN . Names cannot be too long.

Reply:
another downside of the arch and module is that people would not usually know what arch was used for a task. They just want a specific module and don't care what arch that uses. Which is why we should try to only have one candidate for a specific task. Maybe it should be aparcDKT , cerebellum, olfactorybulb, hypothalamus, choroidplexus, piriformcortex etc.

Reply:
Sounds good. Could also just have a default and for people who want to try it out can change it (depending if there actually are different architectures for one submodule).

Reply:
Naming of modules should be consistent with output segmentation file names (so aseg+aparcDKTatlas)